src/app/functions/server/onerep.ts

```diff
@@ -154,6 +154,23 @@ export async function activateProfile(profileId: number): Promise<void> {
   }
 }
 
+export async function deactivateProfile(profileId: number): Promise<void> {
+  const response: Response = await onerepFetch(
+    `/profiles/${profileId}/deactivate`,
+    {
+      method: "PUT",
+    }
+  );
+  if (!response.ok) {
+    log.info(
+      `Failed to deactivate OneRep profile: [${response.status}] [${response.statusText}]`
+    );
+    throw new Error(
+      `Failed to deactivate OneRep profile: [${response.status}] [${response.statusText}]`
+    );
```

Comment:
I think we use this pattern a bunch of places already, so more of a random question; but does it make sense to move the duplicated message into a temp var and use it for the logging and error message?

```suggestion
    const msg = `Failed to deactivate OneRep profile: [${response.status}] [${response.statusText}]`;
    log.info(msg);
    throw new Error(msg);
```

Reply:
Maybe we should consider an error function?